### PR TITLE
Add SigLIP model

### PR DESCRIPTION
**Pull request by as-suvorov**
Ticket: CVS-130640

### CODE REVIEW

File: notebooks/281-siglip-zero-shot-image-classification/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Zero-shot Image Classification with SigLIP
+
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/281-siglip-zero-shot-image-classification/281-siglip-zero-shot-image-classification.ipynb)
+
+Zero-shot image classification is a computer vision task to classify images into one of several classes, without any prior training or knowledge of the classes.
```
Comment (tsavina) [inline notebooks/281-siglip-zero-shot-image-classification/README.md:5]:
```suggestion
Zero-shot image classification is a computer vision task with the goal to classify images into one of several classes without any prior training or knowledge of these classes.
```

File: notebooks/281-siglip-zero-shot-image-classification/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+![zero-shot-pipeline](https://user-images.githubusercontent.com/29454499/207773481-d77cacf8-6cdc-4765-a31b-a1669476d620.png)
+
+In this tutorial, you will use [SigLIP](https://huggingface.co/docs/transformers/main/en/model_doc/siglip) model to perform zero-shot image classification.
```
Comment (tsavina) [inline notebooks/281-siglip-zero-shot-image-classification/README.md:9]:
```suggestion
In this tutorial, you will use the [SigLIP](https://huggingface.co/docs/transformers/main/en/model_doc/siglip) model to perform zero-shot image classification.
```

File: notebooks/281-siglip-zero-shot-image-classification/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Notebook Contents
+
+This tutorial demonstrates how to perform zero-shot image classification using the open-source SigLIP model. The SigLIP model was proposed in [Sigmoid Loss for Language Image Pre-Training](https://arxiv.org/abs/2303.15343) by Xiaohua Zhai, Basil Mustafa, Alexander Kolesnikov, Lucas Beyer. SigLIP proposes to replace the loss function used in [CLIP](https://github.com/openai/CLIP) (Contrastive Language–Image Pre-training) by a simple pairwise sigmoid loss. This results in better performance in terms of zero-shot classification accuracy on ImageNet.
```
Comment (tsavina) [inline notebooks/281-siglip-zero-shot-image-classification/README.md:13]:
```suggestion
This tutorial demonstrates how to perform zero-shot image classification using the open-source SigLIP model. The SigLIP model was proposed in the [Sigmoid Loss for Language Image Pre-Training](https://arxiv.org/abs/2303.15343) paper. SigLIP suggests replacing the loss function used in [CLIP](https://github.com/openai/CLIP) (Contrastive Language–Image Pre-training) with a simple pairwise sigmoid loss. This results in better performance in terms of zero-shot classification accuracy on ImageNet.
```

File: notebooks/281-siglip-zero-shot-image-classification/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+1. Instantiate model
+1. Run PyTorch model inference
+1. Convert model to OpenVINO Intermediate Representation (IR) format.
+1. Run OpenVINO model
+1. Apply post-training quantization using NNCF
+    1. Prepare dataset
+    1. Quantize model
+    1. Run quantized OpenVINO model
+    1. Compare File Size
+    1. Compare inference time of the FP16 IR and quantized models
```
Comment (tsavina) [inline notebooks/281-siglip-zero-shot-image-classification/README.md:32]:
```suggestion
The notebook contains the following steps:

1. Instantiate model.
1. Run PyTorch model inference.
1. Convert the model to OpenVINO Intermediate Representation (IR) format.
1. Run OpenVINO model.
1. Apply post-training quantization using [NNCF](https://github.com/openvinotoolkit/nncf):
    1. Prepare dataset.
    1. Quantize model.
    1. Run quantized OpenVINO model.
    1. Compare File Size.
    1. Compare inference time of the FP16 IR and quantized models.
```

File: notebooks/281-siglip-zero-shot-image-classification/README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+NNCF performs quantization within the OpenVINO IR. It is required to run the first notebook before running the second notebook.
```
Comment (tsavina) [inline notebooks/281-siglip-zero-shot-image-classification/README.md:34]: which one is first which one is second? better use their names here

File: notebooks/281-siglip-zero-shot-image-classification/README.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+We will use SigLIP model for zero-shot image classification. The result of model work demonstrated on the image below
```
Comment (tsavina) [inline notebooks/281-siglip-zero-shot-image-classification/README.md:36]:
```suggestion
The results of the SigLIP model's performance in zero-shot image classification from this notebook are demonstrated in the image below.
```

File: notebooks/281-siglip-zero-shot-image-classification/README.md (hunk 3 above)
Comment (tsavina) [reply notebooks/281-siglip-zero-shot-image-classification/README.md:13]: seems unnecessary

**Comment by as-suvorov** [comment]
<p>I wanted to highlight that context provided with label can significantly impact on model performance. From the second glance I think it can be irrelevant for real use case. Will remove one of the labels.</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1649/discussion/#comment-1910244548'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by as-suvorov** [comment]
<p>Label removed</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1649/discussion/#comment-1910244548'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by as-suvorov** [comment]
<p>Moved gradio to the end. Switched to quantized model</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1649/discussion/#comment-1910248390'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by as-suvorov** [comment]
<p>Thanks for the catch. It works in vs code and I was sure links are not case sensitive. Fixed</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1649/discussion/#comment-1912102791'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by as-suvorov** [comment]
<p>Parent dir creation removed</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1649/discussion/#comment-1912102818'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by eaidova — APPROVED** [review]

**Review by aleksandr-mokrov — APPROVED** [review]

**Pull request merged**